tests/sentry/api/endpoints/test_project_group_index.py

```diff
@@ -466,7 +466,7 @@ def test_self_assign_issue_next_release(self):
         assert response.status_code == 200
         assert response.data['status'] == 'resolved'
         assert response.data['statusDetails']['inNextRelease']
-        assert response.data['assignedTo']['id'] == six.text_type(self.user.id)
+        assert response.data['assignedTo'] == six.text_type(u'user:{}'.format(self.user.id))
```

Comment:
so, this response serialization format is the nasty part that could break people's assumptions about our API